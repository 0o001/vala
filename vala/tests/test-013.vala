using GLib;

class Maman.Bar {
	static int main (int argc, string[] argv) {
		stdout.printf ("Break Test: 1");
		
		int i;
		for (i = 0; i < 10; i++) {
			stdout.printf (" 2");
			break;
		}
		
		stdout.printf (" 3\n");
		
		return 0;
	}
}
