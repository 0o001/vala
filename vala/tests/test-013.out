Break Test: 1 2 3
